includes/class-wc-tracker.php

```diff
@@ -345,7 +362,7 @@ public static function get_product_counts() {
 
 		$product_statuses = get_terms( 'product_type', array( 'hide_empty' => 0 ) );
 		foreach ( $product_statuses as $product_status ) {
-			$product_count[ $product_status->name ] = $product_status->count;
+			$product_count[ 'product_type_' . str_replace( array( ' ', '-' ), '_', strtolower( $product_status->name ) ) ] = $product_status->count;
```

Comment:
I'm all up for normalization, but I think this can theoretically create collisions. I guess we could use slug instead, those can't be the same I think, what do you reckon?

Additionally, wouldn't this create a break in the data, where previously it would previously use key e.g. "Variable Subscription", while now it would end up being "product_type_variable_subscription", so this would require a change in the data layer if there are any reports etc, no? 

Have you checked with folks accepting the data? I think the receiving end can actually flatten arrays so we could use send sth like 
```
$blog_details => {
  product_counts => {
    simple => 3,
    variable => 10,
...
  }
}
```
and it would be represented nicely, but I guess best to check with folks from woo data team?

Reply:
> I guess we could use slug instead, those can't be the same I think, what do you reckon?

So Tracks does not accept spaces or hyphens.

> Additionally, wouldn't this create a break in the data, where previously it would use key e.g. "Variable Subscription", while now it would end up being "product_type_variable_subscription", so this would require a change in the data layer if there are any reports etc, no?

Not sure what you mean here? Can you explain more?

> Have you checked with folks accepting the data?

So I tagged the person who requested this data and they have approved it so I didn't think otherwise.

Reply:
> Tracks does not accept spaces or hyphens

Interesting, didn't know! 

> Not sure what you mean here? Can you explain more?

`get_product_counts` is now shared between Tracks and Woo Tracker. So I'm wondering if we make this translation now, is it fine for both? I can see we call this function also from `WC_Tracker::get_tracking_data`, and that would change the data we send to Woo Tracker, right?

> So I tagged the person who requested this data and they have approved it so I didn't think otherwise.

True. I thought we should check with Woo data folks who receive the tracker data as well.